spec/Body/Body.vspec

```diff
@@ -90,11 +90,102 @@ Windshield.Wiping:
+  
+Windshield.Wiping.UserControl.RequestedIntensity:
+  datatype: uint8
+  type: actuator
+  description: Relative intensity/sensitivity for interval and rain sensor mode as requested by user/driver.
+               0 - wipers inactive.
+               1 - minimum intensity (lowest frequency/sensitivity, longest intervals).
+               2/3/4/... - higher intensity (higher frequency/sensitivity, shorter interval).
+               Maximum value supported is vehicle specific.
+
+
+Windshield.Wiping.SystemControl:
+  type: branch
+  description: Low Level signals to control behavior of wiper system in detail.
+               These signals are typically not directly available to the user/driver of the vehicle.
+
+
+Windshield.Wiping.SystemControl.Mode:
+  datatype: string
+  type: actuator
+  allowed: [
+    STOP, # Wipers finish current wiping cycle and then moves to parking position
+    STOP_APP, # Wipers finish current wiping cycle and then moves to alternative parking position
+    PLANT_MODE, # Wiping disabled. Exact behavior is vehicle specific
+    EMERGENCY_STOP, # Wiping immediately stopped
+    SLEEP_MODE, # System in low energy consumption mode, e.g. bus communication off
+    GOTO_POS, # Wipers moves to position specified by "Position" and then stops
+    PERMANENT_WIPE, # Wipers moves permanently with speed specified by "Frequency"
+    PREDRIVE_CHECK # Wiper system is request to perform self check
+    ]
+  description: Requested mode of wiper system.
+  comment: On this level interval and rainsensor functionality is not present, they are instead
+           supposed to be managed by alternating between PERMANENT_WIPE and STOP/STOP_APP.
+
+Windshield.Wiping.SystemControl.Frequency:
+  datatype: uint8
+  unit: rpm
+  type: actuator
+  description: Wiping frequency/speed, measured in cycles per minute.
+               The signal concerns the actual speed of the wiper blades when moving.
+               Intervals/pauses are excluded, i.e. the value corresponds to the number of cycles
+               that would be completed in 1 minute if wiping permanently.
+  comment: Examples - 0 = Wipers stopped, 80 = Wipers doing 80 cycles per minute (in permanent mode).
+
+Windshield.Wiping.SystemControl.Position:
+  datatype: float
+  unit: degrees
+  type: actuator
+  min: -180
+  max: 180
+  description: Position of wiper blades relative to default parking position.
+               Default parking position shall have the position 0 degrees.
+               Location of default parking position and direction of positive/negative degrees is vehicle specific.
+
+
+Windshield.Wiping.SystemControl.IsFailSafeModeActive:
+  datatype: boolean
+  type: actuator
+  description: Specifies behavior if communication is lost.
+               True = Fail Safe Mode Active, wipers may e.g. start automatically if communication is lost.
+               False = False Safe Mode Inactive.
+
+
+Windshield.Wiping.SystemControl.Load:
+  datatype: uint16
+  unit: W
```

Comment:
Watt currently not supported - fixed by https://github.com/COVESA/vss-tools/pull/134